azurerm/resource_arm_app_service.go

```diff
@@ -679,6 +707,14 @@ func flattenAzureRmAppServiceMachineIdentity(identity *web.ManagedServiceIdentit
 		result["tenant_id"] = *identity.TenantID
 	}
 
+	identityIds := make([]string, 0)
+	if identity.UserAssignedIdentities != nil {
```

Comment:
I think you could use the helper function `utils.FlattenStringSlice` here?

Reply:
Sorry, I wasn't able to use this function `utils.FlattenStringSlice`.